flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -168,6 +168,11 @@
 	public static final String LIBRARY_CACHE_MANAGER_CLEANUP_INTERVAL = "library-cache-manager.cleanup.interval";
 
 	/**
+     * The config parameter defining the task manager resource profile.
+	 */
+	public static final String TASK_MANAGER_RESOURCE_PROFILE_KEY = "taskmanager.resourceProfile";
```

Comment:
Yes, it is a internal config. I though flink put all config consts here and use @PublicEvolving to identify the ones can be used by users. Do you have any suggestion for where to put the internal configs?